pkg/util/accounthelper/accounthelper.go

```diff
@@ -0,0 +1,153 @@
+package accounthelper
```

Comment:
The refactoring into this file is much more than I was looking for and fundamentally I don't think it takes us in the right direction.  It doesn't feel like a good abstraction.  Can you largely revert this bit?  I think we're better keeping a bit of separation between plugin code and external containers - I definitely don't like moving bits of Initialize() in here.  I am happy for the etcdbackup container to do `c := bsc.GetContainerReference(EtcdBackupContainer); c.CreateIfNotExists(nil)` on its own if you want that.  All I'm really looking for is a lightweight way to share (not copy/paste) the code that was in `newAzureClients`.